### PR TITLE
indexer-v2: implement get_epochs, validator-apys, and get_stakes_by_ids

### PR DESCRIPTION
**Pull request by bmwill**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by stefan-mysten — APPROVED** [review]
Looks good. Any plans to add stake subsidy related functionality? Or it will go in another PR?

**Comment by bmwill** [comment]
@stefan-mysten I'm not sure i understand your ask

File: crates/sui-indexer/src/apis/governance_api_v2.rs [hunk 1]
```diff
@@ -52,12 +261,13 @@ impl GovernanceReadApiServer for GovernanceReadApiV2 {
...
-        // Need Dynamic field queries
-        unimplemented!()
+        self.get_stakes_by_ids(staked_sui_ids)
+            .await
+            .map_err(Into::into)
     }
 
     async fn get_stakes(&self, owner: SuiAddress) -> RpcResult<Vec<DelegatedStake>> {
-        // Need Dynamic field queries
+        // Need object_type column in objects table
```
Comment (wlmyng) [inline crates/sui-indexer/src/apis/governance_api_v2.rs:270]: What about 
`select * from objects where df_object_type=’0x3::staking_pool::StakedSui’;`

**Review by wlmyng — APPROVED** [review]

File: crates/sui-indexer/src/apis/governance_api_v2.rs (hunk 1 above)
Comment (stefan-mysten) [reply crates/sui-indexer/src/apis/governance_api_v2.rs]: Thanks for adding the `get_stakes` function!

**Pull request merged**